packages/patternlab/styleguide/source/_patterns/02-molecules/button-dropdown.twig

```diff
@@ -1,18 +1,31 @@
 <div class="ma__button-dropdown-wrapper js-button-dropdown">
+  {% set menuId = button.menuId ?? 'menu-' ~ random() %}
   {% set buttonClass = 'ma__button-dropdown js-button-dropdown-toggle' %}
   {% include "@atoms/01-buttons/button.twig" with {
     button: button|merge({
-      'class': 'ma__button-dropdown js-button-dropdown-toggle'
+      'class': buttonClass,
+      'aria-haspopup': 'true',
+      'aria-controls': menuId
     })
-   } 
+  }
   %}
   <div class="ma__button-dropdown__subitems js-button-dropdown-content is-closed">
-    <ul id="{{ button.menuId }}" role="menu" aria-labelledby="{{ button.id }}" class="ma__button-dropdown__subitems-container">
+    {% if button.items|length > 1 %}
```

Comment:
Sorry I should be more clear - but I think it's a different `<ul>` not the one in the dropdown but outside in the page header

Reply:
I meant the list of buttons in the page header
![Screenshot 2025-02-24 at 10 38 44 AM](https://github.com/user-attachments/assets/ac65b521-8cc2-4521-9676-ede9620ae659)

Maybe we could also update this page to reflect the new changes - replacing the multiple buttons with the button dropdown component https://mayflower.digital.mass.gov/patternlab/b/patternlab/DP-36147_login_links/?p=viewall-organisms-page-header

Reply:
I think the dropdown could remain a list even if there's only one item because it's being announced as a menu - so you could revert this change

Reply:
Changed Page header and fixed A11y issue.